pkg/util/cluster.go

```diff
@@ -59,6 +61,13 @@ func (c *ClusterInfoImp) Init(ctx context.Context, cl client.Client, logger logr
 		err = c.initKubernetes(cl)
 	}
 
+	varValue, varExists := os.LookupEnv(KvUiPluginImageEnvV)
+	if !varExists || varExists && len(varValue) > 0 {
+		c.consolePluginImageProvided = true
+	} else {
+		c.consolePluginImageProvided = false
+	}
```

Comment:
```suggestion
	c.consolePluginImageProvided = !varExists || varExists && len(varValue) > 0
```

Reply:
actually the logic wasn't quite accurate.
we want to set this flag to `true` if and only if both the var exists and its value has a non-zero length.
thanks